README.md

```diff
@@ -66,13 +66,13 @@ Grab via Maven:
 <dependency>
   <groupId>io.voucherify.client</groupId>
   <artifactId>voucherify-java-sdk</artifactId>
-  <version>10.0.0</version>
+  <version>10.1.0</version>
```

Comment:
thinking loud... moze wprowadzimy juz globalnie zasade ze podbijanie wersji to nie w PRkach, tylko jako osobna czynnosc w https://docs.google.com/document/d/1BcgHqO6XE017Dl-fyxheBdp_DpQcnMX98tukGWXVgA4/edit#
?

Reply:
można tak zrobić

Reply:
ok, to dodalem stepy w tym dock-u